firmware/hw_layer/adc/adc_inputs.cpp

```diff
@@ -463,32 +463,11 @@ static void configureInputs(void) {
 
 	addChannel("Baro Press", engineConfiguration->baroSensor.hwChannel, ADC_SLOW);
 
-	addChannel("TPS 1 Primary", engineConfiguration->tps1_1AdcChannel, ADC_SLOW);
```

Comment:
I do not get it - do we just remove these lines?

Reply:
They're replaced by the call added in adc_subscription.cpp. Since the slow ADC no longer uses the conversion sequence computed at runtime (it just converts every channel), there's no need to manually initialize every analog channel here.